workspace_host/interface.js

```diff
@@ -710,6 +711,7 @@ async function _syncInitialZipAsync(workspace) {
 const _syncInitialZip = util.callbackify(_syncInitialZipAsync);
 
 function _syncPullContainer(workspace, callback) {
```

Comment:
```suggestion
function _getInitialFiles(workspace, callback) {
```

I think this is getting the files, not the container?